web/services/models.py

```diff
@@ -65,14 +65,18 @@ def notify_report_received(sender, instance, signal, *args, **kwargs):
     :param args: Unused
     :param kwargs: Unused
     """
-    if instance.user is None:
```

Comment:
@DanielMurphy22 I think this change has called the test failure. When `instance.user` is `None` then you'll get a problem with doing `instance.user.email` later.